.github/workflows/test.yml

```diff
@@ -0,0 +1,21 @@
+name: Run pytest
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python: ["3.9"]
```

Comment:
I'm not sure which versions of Python we run in production, but this would make it pretty easy to tests against multiple different versions (e.g. if we have different versions running on AdminBot, or want to upgrade some lambdas to a new version, etc.).